R/functions-utils.R

```diff
@@ -0,0 +1,49 @@
+#' Helper function to check the data type even if `x` is an `Rle`.
+#'
+#' @param x
+#'
+#' @param class2 `character(1)` specifying the class for which should be tested.
+#'
+#' @author Johannes Rainer
+#'
+#' @noRd
+.is_class <- function(x, class2) {
```

Comment:
`MsCoreUtils`?

Reply:
I almost added this function to `MsCoreUtils` already, but then I decided to not do that:
- this is really just a very simple helper that works in our specific case - I think other people using `MsCoreUtils` might be confused by what this function does.
- what name to use? `isClass` is not possible, maybe `isClassRle`?

I didn't want to add too many functions to `MsCoreUtils`, but if you think it would make sense I could add it.

Reply:
Was just a question. We can keep it here.

Reply:
OK, then let's keep that one here.